src/engine/bank03.asm

```diff
@@ -2712,27 +2712,62 @@ Func_d244: ; d244 (3:5244)
 	farcall Func_80ba4
 	jp IncreaseScriptPointerBy2
 
-Func_d24c: ; d24c (3:524c)
-	ld hl, $525e
+ScriptCommand_ShowMultichoiceTextbox_ChooseDeckToDuelAgainst: ; d24c (3:524c)
+	ld hl, .multichoice_menu_args
 	xor a
-	call Func_d28c
+	call ShowMultichoiceTextbox
 	ld a, [wd695]
 	ld c, a
 	set_flag_value EVENT_FLAG_76
 	jp IncreaseScriptPointerBy1
 
-	INCROM $d25e, $d271
+.multichoice_menu_args ;d25e
+	dw $0000 ;NPC title for textbox under menu
```

Comment:
A note on style, we typically put a space after `;` at the start of a comment.

Reply:
Fixed in the latest commit